src/main/java/frc/robot/HwPneumatics.java

```diff
@@ -5,7 +5,7 @@
 import edu.wpi.first.wpilibj.Compressor;
 
 public class HwPneumatics {
-    public Compressor c = new Compressor(0);
+    public Compressor c = new Compressor(0);//needs this so we can actually use pneumatics
```

Comment:
I am reasonably certain that you do not actually need this to run the compressor, it should just run automatically. You only need this if we want to manually control when the compressor does/does not run.